game-core/src/main/java/tools/map/making/ConnectionFinder.java

```diff
@@ -61,21 +61,19 @@
   // default 32, or if LINE_THICKNESS is given 16 x linethickness
   private double minOverlap = 32.0;
 
-  private ConnectionFinder() {}
-
   /**
    * Runs the connection finder tool.
    *
    * @throws IllegalStateException If not invoked on the EDT.
    */
-  public static void run(final String[] args) {
+  public static void run() {
     checkState(SwingUtilities.isEventDispatchThread());
 
-    new ConnectionFinder().runInternal(args);
+    new ConnectionFinder().runInternal();
   }
 
-  private void runInternal(final String[] args) {
-    handleCommandLineArgs(args);
+  private void runInternal() {
```

Comment:
Codacy found an issue: [The method 'runInternal()' has an NPath complexity of 2643840, current threshold is 200](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6179649)